packages/perseus/src/widgets/grapher/grapher.tsx

```diff
@@ -513,7 +509,7 @@ class Grapher extends React.Component<Props> implements Widget {
     };
 
     showHairlines: (arg1: Coord) => void = (point) => {
-        if (this.props.apiOptions.isMobile && this.props.markings !== "none") {
```

Comment:
This isn't passed through the `transform` functions, so I think it will always be `undefined`.